routes/bookmarks.js

```diff
@@ -0,0 +1,35 @@
+import express from 'express';
+import BookmarksController from '../controllers/BookmarksController';
+import bookmarkValidate from '../middlewares/bookmarksValidate';
+import tokenDecoded from '../middlewares/tokenDecoded';
+// import checkAuthentication from '../middlewares/checkAuthentication';
+
+const router = express.Router();
+
+router.get(
+  '/',
+  // checkAuthentication,
+  tokenDecoded,
```

Comment:
Your authentication middleware should be the same with what the rest of the team uses. Do check-in with the team.

Reply:
will revert to using the checkAuthentication immediately @Chidi3b3r3

Reply:
done